proposed/container.md

```diff
@@ -97,9 +97,9 @@ into the same container (or another container) instead of the delegate container
 The interfaces and classes described as well as relevant exception are provided as part of the
 [psr/container](https://packagist.org/packages/psr/container) package. (still to-be-created)
 
-Packages providing a psr container implementation should declare that they provide psr/container-implementation 1.0.0
+Packages providing a psr container implementation should declare that they provide `psr/container-implementation` `1.0.0`.
```

Comment:
Well, at least the sentence should have a `.` at the end, that was the original reason for a change. After adding a `.` it would be `1.0.0.` and I thought it would be better to separate the version string from the last `.` as they do not belong together.
And as the package name and version is something that will be put into a composer.json file, I'd consider it as formatted code.